modules/orientdb/src/test/java/org/testcontainers/containers/OrientDBContainerWithClassRuleTest.java

```diff
@@ -0,0 +1,27 @@
+package org.testcontainers.containers;
+
+import com.orientechnologies.orient.core.db.ODatabaseSession;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OrientDBContainerWithClassRuleTest {
```

Comment:
I don't think we need a dedicated test for the classrule, it is already tested by the core module, and such tests will only increase the execution time of our CI

Reply:
removed